packages/react-native/src/primitives/Radio/Radio.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useMemo } from 'react';
+import { Pressable, View, ViewStyle } from 'react-native';
+
+import { Label } from '../index';
```

Comment:
To avoid potential circular dependencies, we should import directly from the `Label` directory